### PR TITLE
add test and use constants

### PR DESCRIPTION
**Pull request by hawestra**
add tests for feature attribution drift components and feature importance components
use constants

### CODE REVIEW

**Review by gaugup — APPROVED** [review]

File: model_monitoring/components/feature_attribution_drift/feature_attribution_drift_compute_metrics/spec.yaml [hunk 1]
```diff
@@ -1,15 +1,15 @@
 $schema: http://azureml/sdk-2-0/SparkComponent.json
 type: spark
 
-name: feature_attribution_drift_compute_metrics
-version: 0.3.0
+name: feature_attribution_drift_compute_metrics_spark
```
Comment (mastloui-msft) [inline model_monitoring/components/feature_attribution_drift/feature_attribution_drift_compute_metrics/spec.yaml:4]: The names have changed! No more _spark suffix :)

File: model_monitoring/components/feature_attribution_drift/feature_attribution_drift_compute_metrics/spec.yaml [hunk 2]
```diff
@@ -46,8 +53,3 @@ conf:
         - responsibleai~=0.27.0
         - scikit-learn<1.1
   name: responsibleai-spark
```
Comment (mastloui-msft) [inline model_monitoring/components/feature_attribution_drift/feature_attribution_drift_compute_metrics/spec.yaml:55]: can you ensure the conf is not the last thing in the file? any presence of a bad space (aka new line) will result in Spark creating a new synapse cluster altogether.

File: model_monitoring/components/feature_attribution_drift/feature_attribution_drift_signal_monitor/spec.yaml [hunk 3]
```diff
@@ -1,10 +1,10 @@
 $schema: https://azuremlschemas.azureedge.net/latest/pipelineComponent.schema.json
 type: pipeline
 
-name: feature_attribution_drift_signal_monitor
+name: feature_attribution_drift_signal_monitor_spark
```
Comment (mastloui-msft) [inline model_monitoring/components/feature_attribution_drift/feature_attribution_drift_signal_monitor/spec.yaml:4]: Can you revert the component name changes? :)

File: model_monitoring/components/feature_attribution_drift/feature_attribution_drift_compute_metrics/spec.yaml (hunk 2 above)
Comment (hawestra) [reply model_monitoring/components/feature_attribution_drift/feature_attribution_drift_compute_metrics/spec.yaml:49]: ahh i was wondering why the order of the yaml got changed; updated!

File: model_monitoring/components/feature_attribution_drift/feature_attribution_drift_compute_metrics/spec.yaml [hunk 4]
```diff
@@ -45,9 +45,11 @@ conf:
         - mltable~=1.3.0
         - responsibleai~=0.27.0
         - scikit-learn<1.1
-  name: responsibleai-spark
+    name: responsibleai-spark
+
```
Comment (mastloui-msft) [inline model_monitoring/components/feature_attribution_drift/feature_attribution_drift_compute_metrics/spec.yaml:49]: can you remove the space?

File: model_monitoring/components/feature_attribution_drift/feature_importance_metrics/spec.yaml [hunk 5]
```diff
@@ -46,7 +47,8 @@ conf:
         - mltable~=1.3.0
         - responsibleai~=0.27.0
         - scikit-learn<1.1
-  name: responsibleai-spark
+    name: responsibleai-spark
+
```
Comment (mastloui-msft) [inline model_monitoring/components/feature_attribution_drift/feature_importance_metrics/spec.yaml:51]: remove space

**Review by mastloui-msft — APPROVED** [review]

**Pull request merged**